src/components/PaginationFooter/utils.ts

```diff
@@ -0,0 +1,22 @@
+import type { FooterStyle } from './types';
+import type { BackgroundColors } from '../../hooks/use-extended-theme/types';
+import { COLORING } from '../../utils/color-utils';
+
+const getFooterStyle = (background: BackgroundColors): FooterStyle =>
```

Comment:
seems like a util that needs to be placed in `src/utils` directory. Like the thing we did with `Color Utils`

Reply:
It is not intended for consumption by either `sn-table` or ´sn-pivot-table´. Should we really expose it anyway?

Reply:
if it's for internal use, then it still can be placed in utils but just not exposed IMO

Reply:
yes, exactly. Otherwise every component can have single util modules.
which is kinda happening with constants. 

we can take the same solution and extend it to constants as well. Like having multiple files in constants like:
```jsx
constsnts:
  index.ts // <- all we want to expose
  HeadCellMenuConstans.ts // all headCell menu Constans
  PaginationConstans.ts // all pagination related constants
``` 

mean while if we want to expose something from `HeadCellMenuConstans` or `PaginationConstans` (like the one you mentioned `ELLIPSES` i think), we can import it in index file since that is the single place we expose everything.

by that, we would have all constants across the project only in one single directory of constants which takes care of both public (ones exposed) and private (ones used internally) constants.

Reply:
Alright, will update. I think the important part is that it's super clear what is being exposed as "public" and what is just for consumption within the util repo.